msmbuilder/msm/validation/__init__.py

```diff
@@ -0,0 +1,2 @@
+from __future__ import absolute_import
+from .bootstrapmsm import *
```

Comment:
Don't import `*`
(this pulls in Pool, cpu_count, list_of_1d, MarkovStateModel et al. as well)
(you could define `__all__` in bootstrapmsm.py but why not just import the one class explicitly)